conversion-guide.adoc

```diff
@@ -45,19 +52,28 @@ Soon or later, you'll have to deal with things like encoding a complex tables, e
 
 If you:
 
-. have a file named `image.PNG`
-. use this line to include it `image::image.png[]`
+. have a file named `image.PNG` and
+. use this line: `image::image.png[]` to include it,
 
-This will work on macOS and Windows, but will fail on Linux!
+it will work on macOS and Windows, but will fail on Linux!
 
-Linux uses case-sensitive file systems i.e. `image.PNG` and `image.png` are different file names
+Linux uses case-sensitive file systems, i.e. `image.PNG` and `image.png` are different file names
 and Metanorma will fail to find the specified `image.png`.
 
-If you are on windows, and have a bunch of filenames ending in .PNG, there is a straighforward way to change them all at once:
+If you are on Windows, and have a bunch of filenames ending in .PNG, there is a straightforward way to change them all at once:
 
 1. Go to the folder where the images are located.
 2. Place the cursor in the bar address and write "cmd". The command line will open on that folder location.
-3. On the command line, execute the command `ren *.PNG *.png`, and that's it!
+3. In the command line, execute the command `ren *.PNG *.png`, and that's it!
+
+
+=== AsciiMath vs. LaTeX math
+
+Math expressions can be written in AsciiMath or LaTeX math format. Default option can be determined by using document attribute `:stem: asciimath` or `:stem: latexmath`. In case `:stem:` document attribute is not explicitly written, AsciiMath is applied by default.
+
```

Comment:
@anermina , in this part, I think it is not necessary for the user to write `:stem: asciimath` to set AsciiMath as default. Writting just `:stem:` will do so. (See https://asciidoctor.org/docs/user-manual/#activating-stem-support)

Reply:
I simplified this part. Thanks.